### ISSUE TITLE
Latest changes to iso-random breaks building 

### ISSUE THREAD

**Issue by TheTrunk** [WalletConnect/walletconnect-monorepo#500]
Hi @pedrouid , 
We are using  `@walletconnect/client version 1.4.1` and since yesterday building of our application is crashing due to recent changes in iso-random that walletconnect uses. Version 1.1.0 of iso-random works well, recently released 1.2.0 that is now getting installed with latest wallet connect (1.4.1) breaks the build.

```
    ERROR in ./node_modules/@pedrouid/iso-crypto/dist/cjs/random/index.js 23:31-62
    Module not found: Error: Can't resolve '@pedrouid/iso-random' in '/home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/dist/cjs/random'
    resolve '@pedrouid/iso-random' in '/home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/dist/cjs/random'
      Parsed request is a module
      using description file: /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/package.json (relative path: ./dist/cjs/random)
        Field 'browser' doesn't contain a valid alias configuration
        resolve as module
          /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/dist/cjs/random/node_modules doesn't exist or is not a directory
          /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/dist/cjs/node_modules doesn't exist or is not a directory
          /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/dist/node_modules doesn't exist or is not a directory
          looking for modules in /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/node_modules
            single file module
              using description file: /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/package.json (relative path: ./node_modules/@pedrouid/iso-random)
                no extension
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/node_modules/@pedrouid/iso-random doesn't exist
                .js
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/node_modules/@pedrouid/iso-random.js doesn't exist
                .vue
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/node_modules/@pedrouid/iso-random.vue doesn't exist
                .json
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/node_modules/@pedrouid/iso-random.json doesn't exist
                .css
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/node_modules/@pedrouid/iso-random.css doesn't exist
                .node
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/node_modules/@pedrouid/iso-random.node doesn't exist
            /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-crypto/node_modules/@pedrouid/iso-random doesn't exist
          /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/node_modules doesn't exist or is not a directory
          /home/travis/build/ZelCore-io/ZelCore/node_modules/node_modules doesn't exist or is not a directory
          looking for modules in /home/travis/build/ZelCore-io/ZelCore/node_modules
            single file module
              using description file: /home/travis/build/ZelCore-io/ZelCore/package.json (relative path: ./node_modules/@pedrouid/iso-random)
                no extension
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random is not a file
                .js
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random.js doesn't exist
                .vue
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random.vue doesn't exist
                .json
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random.json doesn't exist
                .css
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random.css doesn't exist
                .node
                  Field 'browser' doesn't contain a valid alias configuration
                  /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random.node doesn't exist
            existing directory /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random
              using description file: /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random/package.json (relative path: .)
                using exports field: ./node/index.js
                  using description file: /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random/package.json (relative path: ./node/index.js)
                    no extension
                      Field 'browser' doesn't contain a valid alias configuration
                      /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random/node/index.js doesn't exist
                    .js
                      Field 'browser' doesn't contain a valid alias configuration
                      /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random/node/index.js.js doesn't exist
                    .vue
                      Field 'browser' doesn't contain a valid alias configuration
                      /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random/node/index.js.vue doesn't exist
                    .json
                      Field 'browser' doesn't contain a valid alias configuration
                      /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random/node/index.js.json doesn't exist
                    .css
                      Field 'browser' doesn't contain a valid alias configuration
                      /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random/node/index.js.css doesn't exist
                    .node
                      Field 'browser' doesn't contain a valid alias configuration
                      /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random/node/index.js.node doesn't exist
                    as directory
                      /home/travis/build/ZelCore-io/ZelCore/node_modules/@pedrouid/iso-random/node/index.js doesn't exist
          /home/travis/build/ZelCore-io/node_modules doesn't exist or is not a directory
          /home/travis/build/node_modules doesn't exist or is not a directory
          /home/travis/node_modules doesn't exist or is not a directory
          /home/node_modules doesn't exist or is not a directory
          /node_modules doesn't exist or is not a directory
     @ ./node_modules/@pedrouid/iso-crypto/dist/cjs/index.js 16:13-32
     @ ./node_modules/@walletconnect/iso-crypto/dist/cjs/index.js 5:39-70
     @ ./node_modules/@walletconnect/client/dist/cjs/index.js 5:39-75
    
    webpack 5.35.1 compiled with 1 error in 284128 ms
 ```
 
 Thank you for providing a fix.

**Comment by gbreux** [comment]
Have you found a workaround while we wait for a fix?

** edit **
https://www.npmjs.com/package/npm-force-resolutions will do the job for now :)

**Comment by pedrouid** [comment]
Thanks for reporting it! I've pushed a fix with iso-random 1.2.1. Can you confirm that is solved?

**Comment by TheTrunk** [comment]
@gbreux As a workaround I was simply using this in my package.json (using yarn as package manager)
```
  "resolutions": {
    "@pedrouid/iso-random": "1.1.0"
  },
  ```

@pedrouid Thank you 1.2.1 resolves it.
